README.md

```diff
@@ -23,10 +23,15 @@ branch is our master branch.
 We are using the [Akari Link Shortener](https://waa.ai/) to create a short URL: [https://waa.ai/ucrsv](https://waa.ai/ucrsv). The
 first three characters "ucr" refers to our school's name, and the last 2 characters "sv" are short for "schedule visualizer". We also have [a QR code](http://bradleycai.github.io/ucr-schedule-visualizer/img/waaai-qr.png).
 
+End of life
+-----------
+
+With UCR's recent switch to Banner, we've decided to stop working on UCRSV. We plan to add minimal compatibility for it's schedule, but that'll be the last thing we do for this website. You can read more about it [here](https://bradleycai.github.io/ucr-schedule-visualizer/about.html#end)
+
 Contributing
 ------
 
-Anyone is welcome to contribute! Suggest a feature in the [issues section](https://github.com/BradleyCai/ucr-schedule-visualizer/issues), fork us, or join our [discord dev server](https://discord.gg/0vTWbxIL45ClG4mV).
+Feel free to pick up where we left off! You can @ us on our [discord dev server](https://discord.gg/0vTWbxIL45ClG4mV) if you have any questions.
```

Comment:
This is the readme though. Won't they know that it's open source if they're on the github page?